audit/projects/k8s-staging-addon-manager/iam.json

```diff
@@ -44,12 +44,6 @@
       ],
       "role": "roles/containerscanning.ServiceAgent"
     },
-    {
-      "members": [
-        "user:spiffxp@google.com"
-      ],
-      "role": "roles/owner"
-    },
```

Comment:
This change is expected due to https://github.com/kubernetes/k8s.io/pull/1859#issuecomment-814430035

This addresses #299 for all k8s-staging-* projects